### ISSUE TITLE
Themeify Tutorial?

### ISSUE THREAD

**Issue by 05M0515** [Birkbjo/OctoPrint-Themeify#69]
Wanting to do more than just change colors.

Things like, replacing icons, altering icon sizes, icon colors, etc.

Found info on LESS files, but still have no idea how to go about creating my own.

Using:
OctoPi/OctoPrint
Themeify (of course)
Custom_Background
TouchUI (separate issue)

Just point in right direction if possible.

Thank you.

**Comment by Birkbjo** [comment]
I will look into creating some sort of tutorial. In the meantime you can use the advanced section and that should be enough for most of your requests. You probably don't need to change less-files etc.

I advise you to look up CSS-selectors. See eg. here: https://www.w3schools.com/cssref/css_selectors.asp . That should give you some overview of CSS-selectors that you can use in the advanced section.

**Comment by 05M0515** [comment]
Kewl! Will do... Thank you.

**Comment by Birkbjo** [comment]
Yeah, grab the selectors that you see in the devtools, then put them into the advanced section in themeify along with the values.

**Comment by prisoner06** [comment]
I'm trying to change the background color of the scrollbars and I can't seem to find the correct element. Or, to be more specific, I can't figure out how to use "-webkit-scrollbar" to specify the correct part of the element. I can see where you added it in the .css but when I try adding:: to the element it does not appear to work.  Any tips?
